test/common/assertSnapshot.js

```diff
@@ -36,11 +36,9 @@ async function assertSnapshot(actual, filename = process.argv[1]) {
 }
 
 async function spawnAndAssert(filename, transform = (x) => x) {
-  // TODO: Add an option to this function to alternatively or additionally compare stderr.
-  // For now, tests that want to check stderr or both stdout and stderr can use spawnPromisified.
   const flags = common.parseTestFlags(filename);
-  const { stdout } = await common.spawnPromisified(process.execPath, [...flags, filename]);
-  await assertSnapshot(transform(stdout), filename);
+  const { stdout, stderr } = await common.spawnPromisified(process.execPath, [...flags, filename]);
+  await assertSnapshot(transform(`${stdout}${stderr}`), filename);
```

Comment:
I was mistaken in my previous PR, this is the actual behavior of `test/message/`:
https://github.com/nodejs/node/blob/3a6116dabbddcabf386ee048d94f546e852489a5/test/message/testcfg.py#L70

Reply:
I think the `test/message` behavior inserts a newline between `stdout` and `stderr`? Or is that unnecessary because `stdout` always ends with a newline?

Reply:
see the line of code above in `testcfg.py` it just concatenates them with no new line